e2e/app-template/template/src/api/database-dump/controllers/database-dump.js

```diff
@@ -0,0 +1,25 @@
+const { createCoreController } = require('@strapi/strapi').factories;
+
+module.exports = createCoreController('api::database-dump.database-dump', ({ strapi }) => {
+  return {
+    async dump(ctx) {
+      const db = strapi.db.getConnection(); // knexjs
+      const metadata = strapi.db.metadata;
+      const promises = [];
+
+      metadata.forEach((contentType) => {
+        const tableName = contentType.tableName;
+
+        if (tableName.startsWith('strapi_')) return;
+
+        console.log('DUMPING TABLE', tableName);
+
+        promises.push(db.from(tableName).del());
+      });
+
+      await Promise.all(promises);
+
+      ctx.send(200);
```

Comment:
If possible we should move this into a service. @Marc-Roig are you up for it? :) Would be good if we'd follow our own patterns/ rules.

Reply:
This is in a service now, figured out how to do it 👀